src/components/query/PropertyQueryEditor.tsx

```diff
@@ -351,7 +355,7 @@ export class PropertyQueryEditor extends PureComponent<Props, State> {
 
     const isAssociatedAssets = isListAssociatedAssetsQuery(query);
     const showProp = !!(!isAssociatedAssets && (query.propertyId || query.assetIds));
-    const properties = showProp ? (asset ? asset.properties : []) : [];
+    const properties = showProp ? stateProps.map(({ id, name }) => ({ id, name, value: id, label: name })) : [];
```

Comment:
```suggestion
    const displayProperties = showProp ? stateProps.map(({ id, name }) => ({ id, name, value: id, label: name })) : [];
```

again just trying to think of some other name to differentiate so we don't need stateprops (which is kind of confusing because react has both state and props)